hack/central-repo/upload-plugins.sh

```diff
@@ -62,6 +62,16 @@ CREATE TABLE IF NOT EXISTS "PluginBinaries" (
 		"URI"                TEXT NOT NULL,
 		PRIMARY KEY("PluginName", "Target", "Version", "OS", "Architecture")
 	);
+
+CREATE TABLE IF NOT EXISTS "PluginGroups" (
+		"Vendor"             TEXT NOT NULL,
+		"Publisher"          TEXT NOT NULL,
+		"GroupName"          TEXT NOT NULL,
+		"PluginName"         TEXT NOT NULL,
+		"Target"             TEXT NOT NULL,
+		"Version"            TEXT NOT NULL,
+		PRIMARY KEY("Vendor", "Publisher", "GroupName", "PluginName", "Target", "Version")
```

Comment:
nit (non-blocking): Should we include a description of the Group here? It might be helpful in showing the `tanzu plugin group search` output.

Reply:
Excellent suggestion!  I didn't think to include a description in the original schema design.
Let's do it in a follow-up as you kindly suggest.